var/da/da_setup_structures/da_scale_background_errors.inc

```diff
@@ -4,17 +4,19 @@ subroutine da_scale_background_errors ( be, it )
    INTEGER,        INTENT(IN)    :: it     ! outer-loop index
 !
    real, allocatable, dimension(:,:) :: v1_val , v2_val , v3_val , &
-                                        v4_val , v5_val
+                                        v4_val , v5_val,           &
```

Comment:
Is this putting variables back in that were removed with f08925ab13dd7194

Reply:
This file is now unchanged.